tests/functional/test_retrieval.py

```diff
@@ -93,3 +126,19 @@ def test_input_shapes(torch_metric) -> None:
 
     with pytest.raises(ValueError, match="`preds` and `target` must be of the same shape"):
         torch_metric(preds, target)
+
+
+# test metrics using top K parameter
+@pytest.mark.parametrize(['torch_metric'], [
+    [retrieval_precision]
+])
```

Comment:
no need to use parametrization if only single input
```suggestion

])
```

Reply:
It's future proof. In the next PR I will add recall:
```
@pytest.mark.parametrize(['torch_metric'], [
        [retrieval_precision],
        [retrieval_recall],
    ])
```